ControllerLogic/TokenStorage.cs

```diff
@@ -0,0 +1,53 @@
+public class TokenStorage : ITokenStorage
+{
+    private readonly Dictionary<string, (long , string)> _tokens = new Dictionary<string, (long , string)>();
```

Comment:
można zrobić krotkę nazwaną, aby było wiadome co jest co czyli (long ExpirationInterval, string User) (jakby cos to Nazwy krotek z dużej litery)